src/index.html

```diff
@@ -15,8 +15,82 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#Home"
+        class="nav__img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="nav__container">
+        <a
+          href="#Apple"
+          class="nav__link is-active"
+        >
+          Apple
+        </a>
+        <a
+          href="#Samsung"
+          class="nav__link"
+        >
+          Samsung
+        </a>
+        <a
+          href="#Smartphones"
+          class="nav__link"
+        >
+          Smartphones
+        </a>
+        <a
+          data-qa="hover"
+          href="#Laptops & Computers"
```

Comment:
The `href` attribute contains spaces (`#Laptops & Computers`). Spaces in `href` attributes can cause issues with navigation. Consider replacing spaces with dashes or removing them altogether, e.g., `#LaptopsComputers`.